assets/js/activitypub-header-image.js

```diff
@@ -23,34 +23,36 @@
 	 * See https://github.com/Automattic/wordpress-activitypub/issues/846
 	 */
 	ImageCropperNoCustomizer = wp.media.controller.CustomizeImageCropper.extend( {
-		doCrop: function( attachment ) {
+		doCrop: function ( attachment ) {
 			var cropDetails = attachment.get( 'cropDetails' ),
 				control = this.get( 'control' ),
 				ratio = cropDetails.width / cropDetails.height;
 
 			// Use crop measurements when flexible in both directions.
 			if ( control.params.flex_width && control.params.flex_height ) {
-				cropDetails.dst_width  = cropDetails.width;
+				cropDetails.dst_width = cropDetails.width;
 				cropDetails.dst_height = cropDetails.height;
 
-			// Constrain flexible side based on image ratio and size of the fixed side.
+				// Constrain flexible side based on image ratio and size of the fixed side.
```

Comment:
this makes no sense

Reply:
...but this means we should go through the code and tweak these things, I think!?!